docs/source/tutorial_bt.rst

```diff
@@ -66,150 +18,115 @@ First, we do the necessary imports and some matplotlib style choices.
+:math:`\tilde{C} = C T`
+has Gramians :math:`\tilde{P}` and :math:`\tilde{Q}` satisfying
+:math:`\tilde{P} = \tilde{Q} = \Sigma = \operatorname{diag}(\sigma_i)`,
+where :math:`\sigma_i` are the Hankel singular values
+(see :doc:`tutorial_lti_systems` for more details).
+Such a realization is called balanced.
+
+The truncation part is based on the controllability and observability energies.
+The controllability energy :math:`E_c(x_0)` is the minimum energy (squared
+:math:`\mathcal{L}_2` norm of the input) necessary to steer the system from the
+zero state to :math:`x_0`.
+The observability energy :math:`E_o(x_0)` is the energy of the output (squared
+:math:`\mathcal{L}_2` norm of the output) for a system starting at the state
+:math:`x_0` and with zero input.
+It can be shown for the balanced realization
+(and same for any other realization)
+that,
+if :math:`\tilde{P}` is invertible,
+then
 
 .. math::
 
-    \begin{align}
-        \widetilde{A} \widetilde{P}
-        + \widetilde{P} \widetilde{A}^T
-        + \widetilde{B} \widetilde{B}^T
-        & = 0, \\
-        \widetilde{A}^T \widetilde{Q}
-        + \widetilde{Q} \widetilde{A}
-        + \widetilde{C}^T \widetilde{C}
-        & = 0, \\
-    \end{align}
+    E_c(x_0) = x_0 \tilde{P}^{-1} x_0, \quad
+    E_o(x_0) = x_0 \tilde{Q} x_0.
 
-such that
-:math:`\widetilde{P} = \widetilde{Q} = \Sigma = \operatorname{diag}(\sigma_i)`,
-where :math:`\sigma_i` are the Hankel singular values.
-Then, taking as basis matrices :math:`V, W \in \mathbb{R}^{n \times r}` the
-first :math:`r` columns of :math:`T` and :math:`S` (possibly after
-orthonormalization), gives a reduced-order model
+Therefore, states corresponding to small Hankel singular values are more
+difficult to reach (they have a large controllability energy) and are difficult
+to observe (they produce a small observability energy).
+In this sense, it is then reasonable to truncate these states.
+This can be achieved by taking as basis matrices
+:math:`V, W \in \mathbb{R}^{n \times r}` the first :math:`r` columns of
+:math:`T` and :math:`S`,
+possibly after orthonormalization,
+giving a reduced-order model
 
 .. math::
 
     \begin{align}
-        \widehat{E} \dot{\widehat{x}}(t)
-        & = \widehat{A} \widehat{x}(t) + \widehat{B} u(t), \\
-        \widehat{y}(t)
-        & = \widehat{C} \widehat{x}(t) + D u(t),
+        \hat{E} \dot{\hat{x}}(t)
+        & = \hat{A} \hat{x}(t) + \hat{B} u(t), \\
+        \hat{y}(t)
+        & = \hat{C} \hat{x}(t) + D u(t),
     \end{align}
 
 with
-:math:`\widehat{E} = W^T E V`,
-:math:`\widehat{A} = W^T A V`,
-:math:`\widehat{B} = W^T B`,
-:math:`\widehat{C} = C V`,
-which satisfies the :math:`\mathcal{H}_\infty` (i.e., induced
-:math:`\mathcal{L}_2`) error bound
+:math:`\hat{E} = W^{\operatorname{T}} E V`,
+:math:`\hat{A} = W^{\operatorname{T}} A V`,
+:math:`\hat{B} = W^{\operatorname{T}} B`,
+:math:`\hat{C} = C V`.
+
+It is known that the reduced-order model is asymptotically stable if
```

Comment:
I think it should also be mentioned that we assume the full order model to be asymptotically stable. Maybe we should do this when the LTI system is introduced in this tutorial.